doc/source/user/numpy-for-matlab-users.rst

```diff
@@ -82,6 +82,8 @@ had to use ``dot`` instead of ``*`` to multiply (reduce) two tensors
 (scalar product, matrix vector multiplication etc.). Since Python 3.5 you
```

Comment:
Above in the intro to this section, I would add in some of the description of `np.matrix` that you deleted above, otherwise the discussion to matrix seems out of the blue.

Maybe something like 
```
 'array' or 'matrix'? Which should I use?
========================================

Historically, NumPy has provided a special matrix type, `np.matrix`, which is a subclass of ndarray which makes binary operations linear algebra operations. You may see it used in some existing code. However, we recommend you do not use `np.matrix`, and instead use plain arrays for the following reasons:
```